greenery/fsm.py

```diff
@@ -175,8 +175,14 @@ def accepts(self, input: Iterable[alpha_type], /) -> bool:
         """
         state = self.initial
         for symbol in input:
-            if ANYTHING_ELSE in self.alphabet and symbol not in self.alphabet:
-                symbol = ANYTHING_ELSE
+            if symbol not in self.alphabet:
+                for actual_symbol in self.alphabet:
```

Comment:
I added code in `accepts` to turn a character into a `frozenset` symbol if required. This could well be removed and instead the requirement be added that all symbols are correctly transformed to be part of the alphabet before being passed to the Fsm class. Since Fsm is not public, this shouldn't be to bad.